utils/tang/files/tang.init

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh /etc/rc.common
+
+START=50
+USE_PROCD=1
+
+. $IPKG_INSTROOT/lib/functions/network.sh
+
+start_service() {
+	KEYS=$(find /usr/share/tang/db -name "*.jw*" -maxdepth 1 | wc -l)
+	if [ -z "${KEYS}" ] || [ "${KEYS}" = "0" ]; then # if db is empty generate new key pair
+		mkdir -p /usr/share/tang/db
+		/usr/sbin/tangd-keygen /usr/share/tang/db
+	fi
+
+	config_load "tang"
+	config_get port	 $1 port "9090"
+
+	procd_open_instance
+	procd_set_param command /usr/sbin/tangd -p ${port} -l /usr/share/tang/db
```

Comment:
```suggestion
	procd_set_param command /usr/sbin/tangd -p "$port" -l /usr/share/tang/db
```

Reply:
Same here

Reply:
according to [SC2086](https://www.shellcheck.net/wiki/SC2086), the variable should be quoted. the same applies to KEYS.

```
KEYS="$(find /usr/share/tang/db -name "*.jw*" -maxdepth 1 | wc -l)"
```